### PR TITLE
check-system-s4u-ssh improvements

### PR DESCRIPTION
**Pull request by martinpitt**
I was looking at that PoC today and trying to understand its details. Here are some simplifications and changes that make it look more like what we want to eventually do. Many thanks to @Gundersanne for your help with this!

### CODE REVIEW

File: test/verify/check-system-s4u-ssh [hunk 1]
```diff
@@ -75,6 +75,7 @@ class TestS4USsh(MachineCase):
         """
         ipa_machine.execute("podman exec freeipa bash -ec '%s'" % script)
 
+
```
Comment (martinpitt) [inline test/verify/check-system-s4u-ssh]: stray empty line; next commit removes it again, let's rebase this away

**Review by Gundersanne — APPROVED** [review]
Very nice :)

**Comment by martinpitt** [comment]
Re-pushed to fix up the stray whitespace, no net change. @Gundersanne , can you please re-approve? Danke!

**Review by Gundersanne — APPROVED** [review]
:100:

**Pull request merged**